<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PressSetting</class>
 <widget class="QDialog" name="PressSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pressSuctionValue">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>140</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pressSuctionRight">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pressSuctionLeft">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pressSuction">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>90</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="injectionTimeValue">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>330</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="injectionTimeLeft">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>330</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="injectionTime">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>280</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="injectionTimeRight">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rationPauseTimeValue">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rationPauseTimeLeft">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rationPauseTime">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rationPauseTimeRight">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>140</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="semiAutoSppedValue">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>330</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="semiAutoSppedRight">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>330</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="semiAutoSppedLeft">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="semiAutoSpped">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
